subiquity/models/network.py

```diff
@@ -13,17 +13,17 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import json
 import logging
 import subprocess
 
 from subiquitycore.models.network import NetworkModel as CoreNetworkModel
 from subiquitycore.utils import arun_command
 
-log = logging.getLogger('subiquity.models.network')
+log = logging.getLogger("subiquity.models.network")
```

Comment:
A fair amount of the diff ends up being just this sort of s/'/"/ - for a lot of the Subiquity code base we're standardized on single quote, and having this sort of change in here makes the logic changes harder to see.

Do you mind if we revert the quote changes and just focus on the rest?

Reply:
My mistake running python black formatter on this before committing. Regenerating this PR now without those formatting changes.